src/resources/postcss/base/typography/_headings.pcss

```diff
@@ -61,9 +61,9 @@
 	.tribe-common-h4 {
 		color: var(--color-text-primary);
 		font-family: var(--font-family-sans-serif);
-		font-size: var(--font-size-5);
+		font-size: var(--font-size-7);
 		font-weight: var(--font-weight-bold);
-		line-height: var(--line-height-2);
+		line-height: var(--line-height-1);
 	}
```

Comment:
let's also add `.tribe-common-h4--alt` with `font-weight` of `regular`, much like `.tribe-common-h3--alt`.